go.mod

```diff
@@ -17,8 +17,8 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/google/uuid v1.6.0
 	github.com/hashicorp/go-azure-helpers v0.70.1
-	github.com/hashicorp/go-azure-sdk/resource-manager v0.20241009.1142232
-	github.com/hashicorp/go-azure-sdk/sdk v0.20241009.1142232
+	github.com/hashicorp/go-azure-sdk/resource-manager v0.20241017.1093842
+	github.com/hashicorp/go-azure-sdk/sdk v0.20241017.1093842
```

Comment:
Updating the `go-azure-sdk` should be done in a separate PR otherwise we end up causing conflicts like has happened here.